checks/system/unix.py

```diff
@@ -361,13 +365,17 @@ def check(self, agentConfig):
             try:
                 memData['physTotal'] = int(meminfo.get('MemTotal', 0)) / 1024
                 memData['physFree'] = int(meminfo.get('MemFree', 0)) / 1024
+                if 'MemAvailable' in meminfo:
+                    memData['physUsable'] = int(meminfo.get('MemAvailable', 0)) / 1024
+                else:
+                    # Usable is relative since cached and buffers are actually used to speed things up.
+                    memData['physUsable'] = memData['physFree'] + memData['physBuffers'] + memData['physCached']
```

Comment:
doesn't look like `physBuffers` and `physCached` are populated until L374-375 . Maybe move this block of code below that?